bigbio/biodatasets/genia_ptm_event_corpus/genia_ptm_event_corpus.py

```diff
@@ -0,0 +1,196 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Post-translational-modiﬁcations (PTM), amino acid modiﬁcations of proteins after translation, are one of the posterior
+processes of protein biosynthesis for many proteins, and they are critical for determining protein function such as its
+activity state, localization, turnover and interactions with other biomolecules. While there have been many studies of
+information extraction targeting individual PTM types, there was until recently little effort to address extraction of
+multiple PTM types at once in a unified framework.
+"""
+
+import os
+from pathlib import Path
+from typing import List, Tuple, Dict
+
+import datasets
+from bigbio.utils import parsing, schemas
+from bigbio.utils.configs import BigBioConfig
+from bigbio.utils.constants import Tasks
+
```

Comment:
Sorry for the late review! Could it be possible to add the latest requirement (`_LANGUAGES = [Lang.EN]`) to this PR as well?
Could be done with:
```python
from bigbio.utils.constants import **Lang**, Tasks
_LANGUAGES = [Lang.EN]
```

Reply:
Sure. I'm first merging master in this feature branch, because `Lang` is not yet present at its current status.